demo/screens/ListSellOrdersScreen.test.js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import NavigationTestUtils from "react-navigation/NavigationTestUtils";
 import { shallow } from "enzyme";
-import ListSellOrdersScreen from "./ListSellOrdersScreen";
+import { ListSellOrdersScreen } from "./ListSellOrdersScreen";
```

Comment:
Oh interesting, what led to this change being needed?

Reply:
Yes, it's an interesting trick. Since `redux` connected component exposes HoC, a way to bypass that and access the component directly is to export the class too. Example:

**Component**
```javascript
export class SomeComponent extends React.Component {...}
export default connect(mapStateToProps, mapDispatchToProps)(SomeComponent);
```
**Test**
```javascript
// Connected HoC
//import SomeComponent from "./SomeComponent";

// Component
import { SomeComponent } from "./SomeComponent";
```
Refs: https://redux.js.org/recipes/writing-tests#connected-components

Reply:
Interesting! Thanks for explaining. Resolving.